tests/lib/rules/no-invalid-this.js

```diff
@@ -87,7 +87,7 @@ function extractPatterns(patterns, type) {
     }));
 
     // Flatten.
```

Comment:
```suggestion
```

With the semantic `flat()` now being used, I think the code is sufficient as documentation here